look-and-feel/react/src/Skeleton/Skeleton.tsx

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable react/jsx-no-useless-fragment */
+/* eslint-disable react/no-array-index-key */
+import "@axa-fr/design-system-look-and-feel-css/dist/Skeleton/Skeleton.scss";
+import { ReactNode } from "react";
+
+type isLoadingAndChildrenProps =
+  | { children: ReactNode; isLoading: boolean }
+  | { children?: undefined; isLoading?: boolean };
+
+export type SkeletonProps = {
+  className?: string;
+  grid: number[][];
+  ariaBusy?: boolean;
+  ariaLabel?: string;
+  maxCols?: number;
+  colGap?: number;
+  rowGap?: number;
+} & isLoadingAndChildrenProps;
+
+export const Skeleton = ({
+  className = "af-skeleton",
+  grid = [],
+  ariaBusy = true,
+  ariaLabel = "Chargement",
+  maxCols = 12,
+  colGap = 16,
+  rowGap = 8,
+  isLoading = true,
+  children,
+}: SkeletonProps) =>
+  isLoading ? (
+    <div
+      className={`${className}-container`}
+      aria-label={ariaLabel}
+      role="status"
+      aria-live="polite"
+      aria-busy={ariaBusy}
+      style={
+        {
+          "--max-cols": maxCols,
+          "--col-gap": `calc(${colGap} / var(--font-size-base) * 1rem)`,
+          "--row-gap": `calc(${rowGap} / var(--font-size-base) * 1rem)`,
+        } as React.CSSProperties
+      }
+    >
+      {grid.map((cols, indexRow) =>
+        cols.map((colSize, indexCol) => (
+          <div
+            key={`${indexRow}-${indexCol}`}
+            style={{ "--col-size": colSize } as React.CSSProperties}
+            className={className}
+          />
+        )),
+      )}
+    </div>
+  ) : (
+    <>{children}</>
```

Comment:
pas besoin du fragment ici 
```suggestion
    children
```

Reply:
j'ai des erreurs sur le type de retour si je ne mets pas de fragment

Reply:
Je pense que c'est lié à ton type `isLoadingAndChildrenProps`, pour moi tu n'as pas a le séparer , car que `isloading `vaux `true`, `false `ou `undefined `tu retourne un `ReactNode`

donc pour moi tu devrais juste déclarer ton type de props comme suis:
```javascript
export type SkeletonProps = {
  className?: string;
  grid: number[][];
  ariaBusy?: boolean;
  ariaLabel?: string;
  maxCols?: number;
  colGap?: number;
  rowGap?: number;
} 

export const Skeleton = ({
  className = "af-skeleton",
  grid = [],
  ariaBusy = true,
  ariaLabel = "Chargement",
  maxCols = 12,
  colGap = 16,
  rowGap = 8,
  isLoading = true,
  children,
}: PropsWithChildren<SkeletonProps>) =>
// ...
```

Reply:
done